includes/class.llms.view.manager.php

```diff
@@ -127,6 +126,25 @@ public function add_menu_items() {
 
 	}
 
+	/**
+	 * Add view manager query args to a given url
+	 *
+	 * @since [version]
+	 *
+	 * @param string|false $href The URL or `false` to use the current `$_SERVER['REQUEST_URI']`.
+	 * @param string       $role The role to view the URL as. Accepts any role defined in the `get_views()` method.
+	 * @param array        $args Additional query arguments to add to the URL.
+	 *
+	 * @return string
+	 */
+	public static function create_url( $href, $role, $args = array() ) {
```

Comment:
What's the reason for this?
And if there's a reason for this, because you need a public static method to be called somewhere else, why it doesn't include:
```
		if ( 'self' === $role ) {
			return remove_query_arg( array( 'llms-view-as', 'view_nonce' ) );
		}
```
I mean if we need this public static method then I think it should completely replace the `get_url()` method, as the piece above is part of the url creation... isn't it?

Reply:
I don't know... The other method felt "broken" so I wanted to recreate it without messing with the other method too much... I don't remember writing most of this code so I dno't know what I was thinking... I'll try better.